geom/wkb_test.go

```diff
@@ -507,14 +439,83 @@ func TestWKBMarshalValid(t *testing.T) {
 }
 
 func TestWKBMarshalEmptyPoint(t *testing.T) {
-	g := geomFromWKT(t, "POINT EMPTY")
-	var buf bytes.Buffer
-	err := g.AsBinary(&buf)
-	expectNoErr(t, err)
-	want := hexStringToBytes(t, "0101000000010000000000f87f010000000000f87f")
-	if !bytes.Equal(want, buf.Bytes()) {
-		t.Logf("want:\n%v", hex.Dump(want))
-		t.Logf("got:\n%v", hex.Dump(buf.Bytes()))
-		t.Errorf("mismatch")
+	for i, tt := range []struct {
+		wkt string
+		hex string
+	}{
+		{
+			"POINT EMPTY",
+			"0101000000010000000000f87f010000000000f87f",
```

Comment:
Nit: It might be illustrative to break these one-liners up into multiple lines similarly to how it is done lower down for `MULTIPOINT`.

Reply:
Good call. WKBs are hard enough to read at the best of times. Fixed (note that I've left the WKBs one single lines for the test at the top of the file, since I autogenerated those ones rather than writing them by hand).